CHANGELOG.md

```diff
@@ -62,6 +62,12 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
 #### exonum-testkit
 
+- `CheckpointDb` and `Testkit` have their `rollback` methods updated (#582):
```

Comment:
`CheckpointDb` is not public, so I don't think we should write about changes in it here.